run-tests.sh

```diff
@@ -158,18 +158,18 @@ ARTIFACTS_DIR="$WORKSPACE/artifacts"
 mkdir -p "$ARTIFACTS_DIR"
 
 create_env_file || exit 1
-docker cp "$TESTS_ENV_FILE" "$TEST_CONTAINER_ID:/opt/app-root/src/tests/env.yaml"
+docker cp "$TESTS_ENV_FILE" "$TEST_CONTAINER_ID:/opt/app-root/tests/env.yaml"
```

Comment:
If we configure the WORKDIR in `base` I think we can fix this and future problems using relative routes , something like 

```
TEST_CONTAINER_ID:tests/tests/env.yaml
```